v2/cmd/wails/internal/commands/build/build.go

```diff
@@ -49,13 +49,19 @@ func AddBuildSubcommand(app *clir.Cli, w io.Writer) {
 	compressFlags := ""
 	command.StringFlag("upxflags", "Flags to pass to upx", &compressFlags)
 
-	// Setup Platform flag
-	platform := runtime.GOOS + "/"
-	if system.IsAppleSilicon {
-		platform += "arm64"
-	} else {
-		platform += runtime.GOARCH
+	defaultPlatform := os.Getenv("GOOS")
+	if defaultPlatform == "" {
+		defaultPlatform = runtime.GOOS
 	}
+	defaultArch := os.Getenv("GOARCH")
+	if defaultArch == "" {
+		if system.IsAppleSilicon {
+			defaultArch = "arm64"
+		} else {
+			defaultArch += runtime.GOARCH
```

Comment:
Just a minor adjustment 
```suggestion
			defaultArch = runtime.GOARCH
```

Reply:
That's weird, I thought I caught that :/